app/routes/adapter-files.js

```diff
@@ -17,8 +16,21 @@ const Router = Express.Router();
 // MAIN ////////////////////////////////////////////////////////////////////////
 // =============================================================================
 
-const cwd = Process.cwd();
-const fileNames = Fs.readdirSync(cwd);
+const cwd = process.env.INIT_CWD;
+let dir = cwd;
+
+if (typeof process.argv[2] === 'string') {
+    const partnerName = process.argv[2];
+    const partnerFolder = partnerName.replace(/\s/g, '')
+        .replace(/([a-z])([A-Z])/g, '$1-$2')
+        .toLowerCase();
+    const htWrapperAdaptersDir = cwd.substring(0,
```

Comment:
For argument splitting, the preferred way is one line per argument kind of like an object. Example:

```javascript
const htWrapperAdaptersDir = cwd.substring(
    0,
    cwd.lastIndexOf('ht-wrapper-adapters') + 'ht-wrapper-adapters'.length
);
```